codalab/rest/users.py

```diff
@@ -110,3 +117,22 @@ def fetch_users():
     usernames.discard('')  # str.split(',') will return '' on empty strings
     users = local.model.get_users(usernames=(usernames or None))
     return UserSchema(many=True).dump(users).data
+
+
+@patch('/users')
+def update_users():
+    """Update users."""
+    if request.user.user_id != local.model.root_user_id:
```

Comment:
Users do so by sending request to `/user` which is defined above as `update_authenticated_user`.